Documentation/operations/system_requirements.rst

```diff
@@ -81,22 +83,80 @@ RancherOS_                 >= 1.5.5
 .. _RancherOS: https://rancher.com/rancher-os/
 
-.. [#centos_foot] CentOS 7 requires a third-party kernel provided by `ElRepo <http://elrepo.org/tiki/tiki-index.php>`_
-    whereas CentOS 8 ships with a supported kernel. Note that some more advanced features may not be available on CentOS 7 even with a third-party kernel. For full details on which kernel config options must be enabled in order to use various features, see the section on :ref:`admin_kernel_version` requirements below.
-
 .. note:: The above list is based on feedback by users. If you find an unlisted
           Linux distribution that works well, please let us know by opening a
           GitHub issue or by creating a pull request that updates this guide.
 
-.. note:: Systemd 245 and above (``systemctl --version``) overrides ``rp_filter`` setting
-          of Cilium network interfaces. This introduces connectivity issues
-          (see :gh-issue:`10645` for details). To avoid that, configure
-          ``rp_filter`` in systemd using the following commands:
 
-          .. code-block:: shell-session
+CentOS 7
+~~~~~~~~
+
+CentOS 7 requires a third-party kernel provided by `ElRepo <http://elrepo.org/tiki/tiki-index.php>`_
+whereas CentOS 8 ships with a supported kernel. Note that some more advanced
+features may not be available on CentOS 7 even with a third-party kernel. For
+full details on which kernel config options must be enabled in order to use
+various features, see the section on :ref:`admin_kernel_version` requirements
+below.
+
+
+``systemd``-based distributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``systemd`` 245 and above overrides the ``rp_filter`` setting of Cilium network
+interfaces. This introduces connectivity issues (see :gh-issue:`10645` for
+details). To avoid that, configure ``rp_filter`` in ``systemd`` using a command
+such as the following:
+
+.. code-block:: shell-session
+
+    echo 'net.ipv4.conf.lxc*.rp_filter = 0' > /etc/sysctl.d/99-override_cilium_rp_filter.conf
+    systemctl restart systemd-sysctl
+
+Keep in mind, however, that some distributions may require additional
+overrides. For example, in Ubuntu 22.04 it's also necessary to edit
+``/etc/sysctl.d/10-network-security.conf``, otherwise this change will get
+undone over time. Please refer to your distribution's documentation for
+the right way to perform this override.
+
+Some distributions additionally need to be configured not to manage "foreign"
+routes. This is the case in Ubuntu 22.04, for example (see :gh-issue:`18706`).
+This can usually be done by setting
+
+.. code-block:: text
+
+   ManageForeignRoutes=no
+   ManageForeignRoutingPolicyRules=no
+
+in ``/etc/systemd/networkd.conf``, but please refer to your distribution's
+documentation for the right way to perform this override.
+
+
+Flatcar
+~~~~~~~
+
+Flatcar is known to manipulate network interfaces created and managed by
+Cilium. This is especially true in the official Flatcar image for AWS EKS, and
```

Comment:
Should this come with a version number to help users understand whether the info is recent or not? (We have CentOS 7, Ubuntu 22.04, or systemd 245+ in the sections above).

Reply:
Well, in previous cases, it's an issue that is known to be limited to a specific version (e.g., CentOS 7) or that is known to start with a specific version (e.g., systemd 245). In this case, all versions of Flatcar have this issue AFAIK.

Our docs don't generally include mentions of when this was valid (other than the Cilium version). If we want to start doing that, it may be best to find a way to include when the page was last updated on the rendered page itself.

Reply:
> it may be best to find a way to include when the page was last updated on the rendered page itself.

This would be a good idea regardless of the version numbers in this PR, I think. I filed https://github.com/cilium/cilium/issues/21847 to track this.